Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    intercom-rails (0.2.29)
+    intercom-rails (0.2.30)
```

Comment:
I was wondering about it as well! After some digging it seems it's considered a good practice, instead: https://github.com/github/gitignore/blob/master/Rails.gitignore#L23-L24